modules/scripts/startup-script/files/running-script-warning.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+FINISH_LINE="startup-script exit status"
+ACTIVE=^activ
```

Comment:
is the missing "e" deliberate?

Reply:
Yes, captures both `activating` and `active`

Reply:
Please document this in a comment.